src/arc.ts

```diff
@@ -89,11 +91,91 @@ export function providerHasConfigUi(provider?: any): any | undefined {
   return provider && provider.isTorus;
 }
 
+function getBiconomy(provider: any): any {
+  console.log("getting biconomy");
+  if (biconomy) {
+    return biconomy;
+  }
+  biconomy = new Biconomy(provider, {
+    dappId: "5e961edab93bbb3218f4ca4d",
+    apiKey: "FnUirn0bX.7e02bb26-8725-45a4-8180-2f67a52871d3",
+    debug: true,
```

Comment:
how the debug flag is manifested ? by console logs ? @tomarsachin2271

Reply:
yes if debug flag is true, logs from sdk will print in console window

Reply:
Biconomy has been tested with portis, fortmtic, torus.. need to test with wallet connect and burner connect wallet.